spring-integration-aws/pom.xml

```diff
@@ -124,12 +124,18 @@
       <version>${spring.integration.version}</version>
       <scope>test</scope>
     </dependency>
-		<dependency>
-			<groupId>org.hamcrest</groupId>
-			<artifactId>hamcrest-all</artifactId>
-			<version>1.3</version>
-			<scope>test</scope>
-		</dependency>
+	<dependency>
+		<groupId>org.hamcrest</groupId>
+		<artifactId>hamcrest-all</artifactId>
+		<version>1.3</version>
+		<scope>test</scope>
+	</dependency>
+    <dependency>
+        <groupId>org.apache.tomcat.embed</groupId>
```

Comment:
In demo there is dependency to jetty server. Is there a particular reason for not using the same here? (Of course jetty version doesn't seem latest there as well)